client/blocks/post-share/test/nudges.jsx

```diff
@@ -71,10 +77,8 @@ describe( 'UpgradeToPremiumNudgePure.render()', () => {
 		PLAN_ECOMMERCE_2_YEARS,
 	].forEach( ( plan ) => {
```

Comment:
I'm not sure if it makes a big difference but Jest provides an API to run a test on multiple cases with [`test.each`](https://jestjs.io/docs/next/api#1-testeachtablename-fn-timeout). May be worth checking out.

Reply:
Sure thing, but it's unrelated, so maybe work for another PR I guess.